templates/paragraphs/paragraph--oe-text-feature-media.html.twig

```diff
@@ -15,6 +15,7 @@
 {{ pattern('text_featured_media', {
   'variant': variant,
   'title': not paragraph.field_oe_title.isEmpty ? content.field_oe_title,
+  'text_title': not paragraph.field_oe_feature_media_title.isEmpty ? content.field_oe_feature_media_title,
```

Comment:
This won't work on update path because `core.entity_view_display.paragraph.oe_text_feature_media.default.yml` only has the `field_oe_feature_media_title` enabled on the install path.